aklite-callback-synchronous/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (C) 2021 Foundries.IO
+#
+# SPDX-License-Identifier: MIT
+#
+version: "3.5"
+
+services:
+  aklite-callback:
+    image: hub.foundries.io/lmp/aklite-callback-synchronous:latest
+    restart: unless-stopped
+    privileged: true
```

Comment:
not sure you need this?

Reply:
I can test w/o, but the volume mappings shouldn't allow access to the /var/ dirs w/o running as root.

Reply:
dropped `privileged` and added `port` info